aws/cloud/master-compliance-test.json

```diff
@@ -526,6 +526,37 @@
                         }
                     ]
                 },
+                {
+                    "masterTestId": "PR-AWS-CLD-AG-008",
+                    "type": "rego",
+                    "rule": "file(api_gateway.rego)",
+                    "masterSnapshotId": [
+                        "TEST_API_GATEWAY_01"
+                    ],
+                    "evals": [
+                        {
+                            "id": "PR-AWS-CLD-AG-008",
+                            "eval": "data.rule.api_gateway_not_configured_with_firewall_v2",
+                            "message": "data.rule.api_gateway_not_configured_with_firewall_v2_err",
+                            "remediationDescription": "Make sure you are following the Cloudformation template format presented <a href='https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/apigateway.html#APIGateway.Client.get_stage' target='_blank'>here</a>",
+                            "remediationFunction": "PR_AWS_CLD_AG_008.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "AWS API Gateway REST API is not configured with AWS Web Application Firewall v2 (AWS WAFv2)",
+                    "description": "AWS API Gateway REST API which is not configured with AWS Web Application Firewall. As a best practice, enable the AWS WAF service on API Gateway REST API to protect against application layer attacks. To block malicious requests to your API Gateway REST API, define the block criteria in the WAF web access control list (web ACL).",
+                    "tags": [
+                        {
+                            "cloud": "AWS",
+                            "compliance": [
+                                "NIST 800"
```

Comment:
@jaiminswan we need full compliance list on each test case.

Reply:
added more compliance